update_copyrights

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+##
+# This script checks all image files in the repository for whether they've been modified or added without updating the file tracking image, sound, and music copyright
+##
+
+import argparse
+import contextlib
+import csv
+from operator import itemgetter
+import os
+from pathlib import Path
+import signal
+from subprocess import check_output
+import sys
+
+def signal_handler(signal, func):
+    print("\nProcessing interrupted.")
+    sys.exit(1)
```

Comment:
It's there to give nicer output than the stack track you'd otherwise get when you do ctrl+c.

Reply:
But it's a bad behavior change from what python does by default for cosmetics. Instead of dying by the signal it just exits so for shells that care about that like bash they'll assume everything is fine and keep going. Not really what you want when you press ctrl-c. If you run this in a loop in bash you cannot exit out of the loop with ctrl-c.

Surely there is a proper way to disable the stack trace or make it look nicer without behavior changes.

Reply:
Removed then.

Reply:
For what it's worth if you want it anyway you can do this:
```python
def my_except_hook(exctype, value, traceback):
  if exctype == KeyboardInterrupt:
    print("\nProcessing interrupted.")
  else:
    sys.__excepthook__(exctype, value, traceback)
sys.excepthook = my_except_hook
```